### ISSUE TITLE
Populate Changes tab of a build from a script

### ISSUE THREAD

**Issue by hoenasgraey** [buildbot/buildbot#5044]
Hello I am desperately seeking for help. I've come completely stuck 

I am using the SVN_watcher.py as an example to get the latest changes from my VCS, Unfortunately the VCS that I am using is not one of the popular ones like git, mercurial or DARCS. More push than poll/pull.
I would like to populate the "CHANGES" tab of a build request with the output of my script
![Screenshot from 2019-10-09 19:02:20](https://user-images.githubusercontent.com/56697999/67040510-8c31dd80-f11b-11e9-9b64-dc0d379a8d81.png)

I've managed to write my own VCS_watcher.py.
But now I don't know how/where to use it, in order to get changes as part of a buildrequest.

Basically I've got a periodic builder, that executes a build every 12 hours, my VCS_watcher.py queries the vcs gets the latest changes and returns a list of "buildbot sendchange --master {MASTERHOST}:{PORT} --auth {USER}:{PASS}   --who {USER} {FILENAMES..}" commands,

Now, I don't know where to run VCS_watcher.py so that any changes are associated with a build request in my periodic builder.

Should I be running it on the buildworker?
Should I be running it on the buildmaster?
Should it be in my master.cfg?
How can I get the changes to be part of a buildrequest for builder my periodic_builder?

Regards
